Source/JavaScriptCore/assembler/AssemblerBuffer.h

```diff
@@ -232,6 +241,24 @@ namespace JSC {
             deallocatePinForCurrentThread();
         }
 
+        ALWAYS_INLINE uint32_t update(uint32_t instruction, uint32_t byteIdx)
```

Comment:
This function was previously private.  Why does it have to be public now?

Reply:
It was public before -- this is the main interface AssemblerBuffer uses to talk to the BufferChecksum.